CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ or miniconda installed cd into the xeus-cpp directory and set setup your environ
 
 ```bash
 cd xeus-cpp
-micromamba create -n xeus-cpp environment-dev.yml
+micromamba create -n xeus-cpp -f environment-dev.yml
```

Comment:
```suggestion
micromamba create -f environment-dev.yml -y
```

Reply:
This change just allows us to offer similar instructions between wasm and non wasm builds.